/usr/obj/ports/orthanc-1.3.0/Orthanc-1.3.0/Resources/ThirdParty/minizip/ioapi.c:(.text+0x6f6):
undefined reference to `fopen64'
libCoreLibrary.a(ioapi.c.o): In function `ftell64_file_func':
/usr/obj/ports/orthanc-1.3.0/Orthanc-1.3.0/Resources/ThirdParty/minizip/ioapi.c:(.text+0x72a):
undefined reference to `ftello64'
libCoreLibrary.a(ioapi.c.o): In function `fseek64_file_func':
/usr/obj/ports/orthanc-1.3.0/Orthanc-1.3.0/Resources/ThirdParty/minizip/ioapi.c:(.text+0x7e3):
undefined reference to `fseeko64'

Index: Resources/ThirdParty/minizip/ioapi.h
--- Resources/ThirdParty/minizip/ioapi.h.orig
+++ Resources/ThirdParty/minizip/ioapi.h
@@ -45,7 +45,7 @@
 #include <stdlib.h>
 #include "zlib.h"
 
-#if defined(USE_FILE32API)
+#if defined(USE_FILE32API) || defined(__OpenBSD__)
 #define fopen64 fopen
 #define ftello64 ftell
 #define fseeko64 fseek
